pkg/spec3/benchmark_serialization_test.go

```diff
@@ -19,6 +20,29 @@ var swaggerDiffOptions = []cmp.Option{cmp.Comparer(func(a spec.Ref, b spec.Ref)
 	return a.String() == b.String()
 })}
 
+func TestOpenAPIV3RoundTrip(t *testing.T) {
+	var fuzzer *fuzz.Fuzzer
+	fuzzer = fuzz.NewWithSeed(1646791953)
+	// Make sure we have enough depth such that maps do not yield nil elements
+	fuzzer.MaxDepth(22).NilChance(0.5).NumElements(1, 7)
+	fuzzer.Funcs(OpenAPIV3FuzzFuncs...)
+	expected := &OpenAPI{}
+	fuzzer.Fuzz(expected)
+
+	j, err := json.Marshal(expected)
+	if err != nil {
```

Comment:
I turned down the nil chance, and ran this test in a loop with a few random seeds, and found something is up with `Paths` that doe snot round trip properly

Reply:
Fixed. Accidentally removed the requirement that paths needed to start with a slash in a refactor.